lib/arpv4.mli

```diff
@@ -28,6 +28,9 @@ module Make (Ethif : V1_LWT.ETHIF) (Clock : V1.CLOCK) (Time : V1_LWT.TIME) : sig
       [t]. *)
   val create: Ethif.t -> t
 
+  val add: t -> Ipaddr.V4.t -> Macaddr.t -> unit
+  (** Add an entry in the ARP table. *)
```

Comment:
what is the failure behaviour? I'd add a policy what to do if there's already an entry for the given ipaddr, `(?if_exists : Replace | Error | Drop)`